examples/compute/instance/front_worker/base-flavor.tf

```diff
@@ -0,0 +1 @@
+../../flavor/main.tf
```

Comment:
lets rename the folder to default_secgroup

Reply:
rename and fix links